docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -784,6 +786,18 @@ If enabled, makes the image read-only.
 
 If ReadOnly is set to `true`, mount a read-write tmpfs on /dev, /dev/shm, /run, /tmp, and /var/tmp.
 
+### `ReloadCmd=`
+
+Add `ExecReload` line to the `Service` that runs `exec` with this value.
```

Comment:
```suggestion
Add `ExecReload` line to the `Service` that runs ` podman exec` with this command in this container.
```

Also maybe we should define what ExecReload does, i.e. it will not be run by default but can be triggered by running `systemctl reload <service>`

Reply:
Done